backend/tests/test_main.py

```diff
@@ -1,11 +1,14 @@
 import unittest
 
-from fun_forecast_backend.main import get_activity_forecast
+from fun_forecast_backend.main import get_activity_forecast, search_location_by_coordinates
 
 
 class TestMainApp(unittest.IsolatedAsyncioTestCase):
 
 
     async def test_get_activity_forecast_endpoint__real_input(self):
         output = await get_activity_forecast(verb="hike", lat=36.8637, long=-78.5324)
-        # self.assertTrue(isinstance(crawler, SERPApiClient))
\ No newline at end of file
+
+
+    async def test_get_location_data_by_lat_long__real_input(self):
+        output = await search_location_by_coordinates(lat=36.8637, long=-78.5324)
```

Comment:
we should probably assert something right? Or is this just checking that the endpoint doesn't throw exception

Reply:
This was just for me to test the function without docker :) We need to mock the response and everything to make the unit test more official